docs/usage/configuration-ttw.rst

```diff
@@ -140,70 +140,113 @@ Zones
 -  Zones sensibles et types de zones
 
 
-Gestion des utilisateurs
-========================
+.. _user-management-section:
+
+Users management
+================
+
+Geotrek-admin relies on `Django authentication and permissions system <https://docs.djangoproject.com/fr/4.2/topics/auth/default/>`_.
```

Comment:
Link to the doc in english: https://docs.djangoproject.com/en/4.2/topics/auth/default/